packages/terra-core-docs/src/terra-dev-site/doc/alert/About.1.doc.mdx

```diff
@@ -67,19 +67,31 @@ import Alert from 'terra-alert';
 ```
 
 ## Accessibility
-Please view the full [Accessibility Guide](./accessibility-guide).
 
-<Notice variant="important" ariaLevel="3">
+For further guidance, please view the full [Accessibility Guide](./accessibility-guide).
 
-#### Accessibility Guidance: Live regions
+<Notice variant="important" ariaLevel="3">
 
-For more, please view the [full accessibility guide for live regions and assistive technologies](./accessibility-guide#assistive-technologies)
+<ul>
+    <li>
+        The `alert` notification type will seize keyboard focus by default when an action element is present. This behavior should only be used when absolutely necessary to interrupt the user's workflow.
```

Comment:
Looks like the markdown backticks don't work in these code blocks. Would a \<code\> element do it?

Reply:
For some reason <code> doesn't display the text so it just looks like this

![image](https://github.com/cerner/terra-core/assets/37750902/4019274e-3edd-458a-8494-2599cee8b9cf)

after messing around with it I'm still unsure why

Reply:
Coming back to this, after changing from html list to markdown, the backticks work again!